config/dnsmasq.conf

```diff
@@ -0,0 +1,24 @@
+interface=provisioning
+except-interface=lo
+bind-dynamic
+dhcp-range=IRONIC_DHCP_RANGE
+enable-tftp
+tftp-root=/tftpboot
+
+dhcp-match=ipxe,175
+# Client is already running iPXE; move to next stage of chainloading
+dhcp-boot=tag:ipxe,http://IRONIC_IP:5280/dualboot.ipxe
```

Comment:
The `5280` change is from another PR, let's maybe keep it there?

Reply:
basically this would depend on that change.  This file doesn't exist whereas the other PR updates the existing setup to 5280.  I think it'd be best to test apply that first then appy this one.  Also we can't use port 80 in kubernetes so this is a hard requirement.